media-types.md

```diff
@@ -39,10 +38,6 @@ This section shows where the OCI Image Specification is compatible with formats
 
 - [application/vnd.docker.container.image.v1+json](https://github.com/docker/docker/blob/master/image/spec/v1.md#image-json-description)
 
-### application/vnd.oci.image.serialization.combined.v1+json
-
-- [layout compatible with docker save/load format](https://github.com/opencontainers/image-spec/blob/master/serialization.md#combined-image-json--filesystem-changeset-format)
```

Comment:
Do we want to maintain backwards-compat docs for these retired media types?  They aren't in this spec anymore, but folks used to the Docker spec may be interested in migration suggestions.